index.html

```diff
@@ -55,5 +59,157 @@ <h2 class="social">LET'S CONNECT</h2>
     </ul>
 </div>
 </section>
+<section id="About">
+    <div class="about-section">
+      <img src="./img/Snapshoot Portfolio.png" class="card-img" />
+      <h4 class="about-head">Tonic</h4>
+      <div class="canopy">
+        <ul class="cano">
+          <li class="tec">CANOPY</li>
+          <li class="tech">Back End Dev</li>
+          <li class="tech">2015</li>
+        </ul>
+        <p class="canopy-txt">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required
+        </p>
+        <ul class="tech1">
+          <li class="tech-t">HTML</li>
+          <li class="tech-t">CSS</li>
+          <li class="tech-j">JavaScript</li>
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
```

Comment:
- [ ] Please I highly suggest implementing hover effects to the button:  `Active`, `Enabled`, `Hover` and `Pressed` per [Figma Design Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=Tg5tjOlgyejotrXq-0). This effect should be implemented on all buttons in the project. 😄 

_This is a screenshot of the button effects required by the Figma Design_

![Screenshot (321)](https://github.com/Danny-imo/my-project/assets/78104544/1d458cb1-3c4c-48bb-b0e0-bca7c60b761d)